orthopairs/runOrthopairs.sh

```diff
@@ -0,0 +1,9 @@
+## Update repo
+git pull
```

Comment:
Adding these lines like we did for orthoinference would be good so wherever the current working directory is, the script always knows its directory context for execution.
```
#!/bin/bash
DIR=$(dirname "$(readlink -f "$0")") # Directory of the script -- allows the script to invoked from anywhere
cd $DIR
```